tests/test_annoy_console.py

```diff
@@ -0,0 +1,33 @@
+# Unit this for annoy
+# Import slogpy from slog
+from slogpy.slog import Slog as slog  # noqa: N813, E402
+
+"""This module contains tests for the Slog class, specifically the annoying method,
+which is responsible for slog annoying messages.
+"""
+
+# (note to self) make sure the count is only once......
+
+def test_annoy_unique_message(capsys, msg="This is a annoy test"):  # noqa: D103
+    """Timestamp checking method
+
+    Given: a single line annoy message
+    When: passed into slog.annoy
+    Then: method will check for annoy message
+    """
```

Comment:
The wording here is hard to follow.  It also doesn't really capture the value proposition of annoy().  The reason a developer should use annoy() is to remind themselves that there is more work to be done and they don't want to forget about it, but they also don't want to be overly annoyed by it.  annoy() does this by only displaying the message once.  Looking at the code, it also only puts it in the log once (for each unique message).